dimod/generators/binpacking.py

```diff
@@ -20,24 +20,15 @@
 from dimod.constrained import ConstrainedQuadraticModel
 from dimod.vartypes import BINARY
 
-__all__ = ['random_bin_packing']
+__all__ = ['bin_packing', 'random_bin_packing']
 
 
-def random_bin_packing(num_items: int,
-                       seed: typing.Optional[int] = None,
-                       weight_range: typing.Tuple[int, int] = (10, 30),
-                       ) -> ConstrainedQuadraticModel:
+def bin_packing(weights: typing.List[int], capacity: int) -> ConstrainedQuadraticModel:
```

Comment:
I also noticed that this has `int`, whereas https://github.com/dwavesystems/dwave-optimization/pull/93 allows a `float`. I think this can be a `float`.

Reply:
Good call, updated the type hints: https://github.com/dwavesystems/dimod/pull/1384/commits/f0fe69d53a57e7d753f21525afd19fc76942aad4